empress/recon_vis/plot_tools.py

```diff
@@ -1,28 +1,37 @@
 """
 plot_tools.py
 Plotting tools using matplotlib
-Modified June 26, 2020
 """
 
 # If matplotlib doesn't pop up a window, force it to use tkinter backend
 # matplotlib.use("tkagg")
 from typing import Union
-
 from matplotlib import pyplot as plt
+from matplotlib.lines import Line2D
+from matplotlib.collections import LineCollection
+from matplotlib.textpath import TextPath
+from matplotlib.patches import PathPatch
+import matplotlib.patheffects as PathEffects
+
+
+from empress.recon_vis.render_settings import *
```

Comment:
Don't `import *`. Instead, use `from from empress.recon_vis import render_settings` and refer to the variables as `render_settings.VAR`.